### PR TITLE
docs(website): improve comparision documentation

### PR DESCRIPTION
**Pull request by pmig**
## 📑 Description
As we still get a lot of feedback about the differences to other solutions I improved the comparison section on our website.

## ✅ Checks
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required

### CODE REVIEW

File: website/docs/04_comparisons/01_helm.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+1. **Package updates**<br/>Glasskube provides the possibility to install packages in `@latest` version which leads to automatic updates or install a package in specific version and explore new versions via the `outdated` command and upgrade them with the `upgrade` command. 
+2. **Dependency management**<br/>Glasskube offers - as you would expect - dependency management out of the box. Multiple packages can require a specific package (e.g. `cert-manager`) being installed and updated in the Kubernetes cluster and its preferred namespace.
+3. **Custom Resources Definition (CRD) changes**<br/>Upgrading CRDs will be taken care of by Glasskube to ensure CRs and it's operators don't get out-of-sync. These updates are performed via server-side apply.
+4. **Cloud-Native architecture**<br/>As helm's architecture is purely client side, as it renders our the templates and apply them via the Kubernetes API. Although releases are stored in Kubernetes Secrets there is no first party server-side component for helm making harder to install packages via the GitOps approach.
```
Comment (christophenne) [inline website/docs/04_comparisons/01_helm.md:19]: The first sentence looks a bit off - two times "as" and "our the templates"

Comment (christophenne) [reply website/docs/04_comparisons/01_helm.md:19]: Second sentence: "making *it* harder..." ?

File: website/docs/04_comparisons/01_helm.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+2. **Dependency management**<br/>Glasskube offers - as you would expect - dependency management out of the box. Multiple packages can require a specific package (e.g. `cert-manager`) being installed and updated in the Kubernetes cluster and its preferred namespace.
+3. **Custom Resources Definition (CRD) changes**<br/>Upgrading CRDs will be taken care of by Glasskube to ensure CRs and it's operators don't get out-of-sync. These updates are performed via server-side apply.
+4. **Cloud-Native architecture**<br/>As helm's architecture is purely client side, as it renders our the templates and apply them via the Kubernetes API. Although releases are stored in Kubernetes Secrets there is no first party server-side component for helm making harder to install packages via the GitOps approach.  
+5. **Kubernetes version upgrade compatibility**<br/>Glasskube tires to make Kubernetes version upgrades as smooth as possible by automatically testing all package (combinations) across multiple Kubernetes versions.
```
Comment (christophenne) [inline website/docs/04_comparisons/01_helm.md:20]: Glasskube *tries*

Comment (pmig) [inline website/docs/04_comparisons/01_helm.md:20]: This section is new

File: website/docs/04_comparisons/01_helm.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+---
+slug: helm
+sidebar_label: With Helm
+title: Glasskube vs Helm Comparison
+description: Discover Glasskube, a fresh Kubernetes package manager challenging Helm's shortcomings. Explore automatic updates, dependency management and much more.
```
Comment (pmig) [inline website/docs/04_comparisons/01_helm.md:5]: The description and the ordered list (second comment) is new the other paragraphs did not change (much), but due to the new file name GitHub doesn't show a meaningful side-by-side view.

File: website/docs/04_comparisons/02_timoni.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+sidebar_label: With Timoni
+title: Glasskube vs Timoni Comparison
+description: Timoni installs and updates cloud-native applications as bundles (OCI images) with a great support for configuration, but it requires users to create its own bundles.
+---
+
+# Glasskube vs Timoni Comparison
+
+[Timoni](https://helm.sh/) was created by [Stefan Prodman](https://github.com/stefanprodan) to improve bundling cloud native applications and their lifecycle management.
+
+Timoni is closer related to helm than Glasskube, but brings some features and new approaches to the table Helm doesn't offer.
```
Comment (christophenne) [inline website/docs/04_comparisons/02_timoni.md:12]: helm should be uppercase

**Review by christophenne — CHANGES_REQUESTED** [review]

File: website/docs/04_comparisons/02_timoni.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+Its bundles are OCI images, so the only way to install these bundles is to publish them first to an OCI registry, which makes additional security features like image (co-)singing possible. 
+Similar to Glasskube it performs garbage collection of orphan resources after an uninstallation operation.
+
+Timoni is in general already more mature than Glasskube, but doesn't focus on providing an easy way of package installation and upgrades and as of now there is also no GUI support planned
```
Comment (kosmoz) [inline website/docs/04_comparisons/02_timoni.md:16]: I think this sentence is too long and has too many "and"s. I think it should be split into two sentences.

File: website/docs/04_comparisons/01_helm.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+That is why we started thinking about and prototyping Glasskube—a *new* vision of what a package manager for Kubernetes could look like.
+
+We are trying to take new approach on multiple features, but want to highlight the 5 most important differences that got either just released, are in development right now or are on our roadmap and will be started soon:
+
+1. **Package updates**<br/>Glasskube provides the possibility to install packages in `@latest` version which leads to automatic updates or install a package in specific version and explore new versions via the `outdated` command and upgrade them with the `upgrade` command. 
+2. **Dependency management**<br/>Glasskube offers - as you would expect - dependency management out of the box. Multiple packages can require a specific package (e.g. `cert-manager`) being installed and updated in the Kubernetes cluster and its preferred namespace.
+3. **Custom Resources Definition (CRD) changes**<br/>Upgrading CRDs will be taken care of by Glasskube to ensure CRs and it's operators don't get out-of-sync. These updates are performed via server-side apply.
```
Comment (kosmoz) [inline website/docs/04_comparisons/01_helm.md:18]: it's -> its

Additionally, I'm not sure if SSA really adds any benefit here

File: website/docs/04_comparisons/01_helm.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+description: Discover Glasskube, a fresh Kubernetes package manager challenging Helm's shortcomings. Explore automatic updates, dependency management and much more.
+---
+
+# Glasskube vs Helm Comparison
+
+In late 2023, we published a [blog post](https://glasskube.eu/en/r/knowledge/5-helm-shortcomings/) about, what we thought, were the biggest shortcomings of [Helm](https://helm.sh/)—undoubtedly the most popular package manager for Kubernetes.
+After receiving *TONS* of feedback from the community we realized that we were not alone in being unsatisfied with the status quo.
+That is why we started thinking about and prototyping Glasskube—a *new* vision of what a package manager for Kubernetes could look like.
+
+We are trying to take new approach on multiple features, but want to highlight the 5 most important differences that got either just released, are in development right now or are on our roadmap and will be started soon:
```
Comment (kosmoz) [inline website/docs/04_comparisons/01_helm.md:14]: I think it sounds better like this: 

```suggestion
We take new approach on multiple features, but want to highlight the 5 most important differences that got either just released, are in development right now or are on our roadmap and will be started soon:
```

File: website/docs/04_comparisons/01_helm.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+
+1. **Package updates**<br/>Glasskube provides the possibility to install packages in `@latest` version which leads to automatic updates or install a package in specific version and explore new versions via the `outdated` command and upgrade them with the `upgrade` command. 
+2. **Dependency management**<br/>Glasskube offers - as you would expect - dependency management out of the box. Multiple packages can require a specific package (e.g. `cert-manager`) being installed and updated in the Kubernetes cluster and its preferred namespace.
```
Comment (kosmoz) [inline website/docs/04_comparisons/01_helm.md:17]: [emdash](https://www.thesaurus.com/e/grammar/em-dash/)

**Review by kosmoz — CHANGES_REQUESTED** [review]

File: website/docs/04_comparisons/01_helm.md [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+
+1. **Package updates**<br/>Glasskube provides the possibility to install packages in `@latest` version which leads to automatic updates or install a package in specific version and explore new versions via the `outdated` command and upgrade them with the `upgrade` command.
+2. **Dependency management**<br/>Glasskube offers—as you would expect—dependency management out of the box. Multiple packages can require a specific package (e.g. `cert-manager`) being installed and updated in the Kubernetes cluster and its preferred namespace.
+3. **Custom Resources Definition (CRD) changes**<br/>Upgrading CRDs will be taken care of by Glasskube to ensure CRs and its operators don't get out-of-sync.
+4. **Cloud-Native architecture**<br/>As Helm's architecture is purely client side, it renders the templates and apply them via the Kubernetes API. Although releases are stored in Kubernetes Secrets there is no first party server-side component for helm making it harder to install packages via the GitOps approach.
```
Comment (christophenne) [inline website/docs/04_comparisons/01_helm.md:19]: apply -> applies

**Review by christophenne — APPROVED** [review]

File: website/docs/04_comparisons/01_helm.md (hunk 9 above)
Comment (pmig) [reply website/docs/04_comparisons/01_helm.md]: thanks, fixed! :-)

File: website/docs/04_comparisons/01_helm.md [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+description: Discover Glasskube, a fresh Kubernetes package manager challenging Helm's shortcomings. Explore automatic updates, dependency management and much more.
+---
+
+# Glasskube vs Helm Comparison
+
+In late 2023, we published a [blog post](https://glasskube.eu/en/r/knowledge/5-helm-shortcomings/) about, what we thought, were the biggest shortcomings of [Helm](https://helm.sh/)—undoubtedly the most popular package manager for Kubernetes.
+After receiving *TONS* of feedback from the community we realized that we were not alone in being unsatisfied with the status quo.
+That is why we started thinking about and prototyping Glasskube—a *new* vision of what a package manager for Kubernetes could look like.
+
+We take new approach on multiple features, but want to highlight the 5 most important differences that got either just released, are in development right now or are on our roadmap and will be started soon:
```
Comment (kosmoz) [inline website/docs/04_comparisons/01_helm.md:14]:
```suggestion
We take a new approach on multiple features, but want to highlight the 5 most important differences that got either just released, are in development right now or are on our roadmap and will be started soon:
```

**Review by kosmoz — CHANGES_REQUESTED** [review]

**Review by kosmoz — APPROVED** [review]

**Pull request merged**